Sources/Testing/Discovery+Macro.swift

```diff
@@ -0,0 +1,39 @@
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2023–2025 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+//
+
+/// The type of the accessor function used to access a test content record.
+///
+/// The signature of this function type must match that of the corresponding
+/// type in the `_TestDiscovery` module. For more information, see
+/// `ABI/TestContent.md`.
+///
+/// - Warning: This type is used to implement the `@Test` macro. Do not use it
+///   directly.
+public typealias __TestContentRecordAccessor = @convention(c) (
```

Comment:
Given that the `_TestDiscovery` module is not publicly imported from the `Testing` module, why do we need to (double) underscore-prefix this and other decls in this module?

Reply:
Look where this file is. :)